app/shared/context/AudioPlayerContext.tsx

```diff
@@ -0,0 +1,50 @@
+'use client';
+
+import { createContext, useCallback, useContext, useMemo, useState } from 'react';
+
+export type AudioPlayerContextType = {
+  src: string;
+  trackName: string;
+  isPlaying: boolean;
+  playTrack: (src: string, trackName: string) => void;
+  togglePlay: () => void;
+};
+
+export const AudioPlayerContext = createContext<AudioPlayerContextType | undefined>(undefined);
+
+export const AudioPlayerProvider = ({ children }: { children: React.ReactNode }) => {
+  const [src, setSrc] = useState(
+    '/api/blob-url?blobName=%D0%9F%D0%BE%D0%B5%D0%BC%D0%B0%20%D0%BF%D1%80%D0%BE%20%D0%BB%D1%96%D1%81&folderName=compositions'
```

Comment:
Why you write here encoded name ?
Our implemented blob storage work with simple names as 'Поема про ліс'
Yes, under the hood it will encode this name, but here you can write just 'Поема про ліс'
```suggestion
    '/api/blob-url?blobName=Поема про ліс&folderName=compositions'
```

Reply:
And if i understand correct so this 'Поема про ліс' composition name its a base song that will run if our user is not on artistry page
So i think you should move this name to constants file and use it from there, because we can use it in different files